src/app/books.ts

```diff
@@ -168,7 +168,7 @@ export const BOOKS : Book[] = [
 		"id": 7,
 		"title": "The Stranger and Her Friend",
 		"edition": "?",
-		"img": "https://cdn-img.fimfiction.net/story/o50s-1432428591-20514-large",
+		"img": "https://cdn-img.fimfiction.net/story/av08-1551073366-20514-full",
```

Comment:
Please use the `-large` version of fimfic images you link in. Full is a much bigger file.

Reply:
Changed to `-large`